templates/pe-client-tools-2019.8.rb.erb

```diff
@@ -0,0 +1,7 @@
+cask 'pe-client-tools-2019.8' do
+<%= source_stanza %>
+  name 'PE Client Tools'
+  homepage "https://puppet.com/docs/pe/2019.8/installing_pe_client_tools.html"
```

Comment:
```suggestion
  homepage "https://puppet.com/docs/pe/latest/installing_pe_client_tools.html"
```

Reply:
This file is explicitly for the named version. I think latest is in the file without a version in the name.